code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -332,6 +325,25 @@
 	source = null
 	return ..()
 
+/obj/item/restraints/legcuffs/beartrap/changeling
+	name = "tentacle mass"
+	desc = "A disgusting mass of flesh wrapped around some poor persons legs."
+	icon_state = "fleshtrap" //Never on ground, only on examine, so doesn't need to be super detaled
+	trap_damage = 5
+	armed = TRUE
+	anchored = TRUE
+	breakouttime = 5 SECONDS
+	cuffed_state = "fleshlegcuff"
+	flags = DROPDEL
+
+/obj/item/restraints/legcuffs/beartrap/changeling/Crossed(AM, oldloc)
+	if(!iscarbon(AM) || !armed)
+		return
+	var/mob/living/carbon/C = AM
+	C.apply_status_effect(STATUS_EFFECT_CLINGTENTACLE)
+	..()
```

Comment:
Please space this out, it's not normal for us to call parent like this so we can at least make it somewhat more readable

Reply:
done!